lib/ui/index.js

```diff
@@ -325,6 +325,7 @@ class UI {
             if (error.logToFile()) {
                 const logLocation = system.writeErrorLog(stripAnsi(`${debugInfo}\n${verboseOutput}`));
                 this.log(`\nAdditional log info available in: ${logLocation}`);
+                this.log(`\nTry running ${(chalk.cyan('ghost doctor'))} to check your system for known issues.`);
```

Comment:
Out of curiosity (sorry I didn't catch this before) - do we want to only output the "ghost doctor" message when the error will log to a file?

For example - any SystemErrors (which Ghost-CLI throws when it runs into certain permissions errors) don't log to a file, so this message won't get output.

Reply:
Good point! I'll put it after the conditional, so it'll always be logged!